action.yml

```
@@ -15,6 +15,9 @@ inputs:
     description: 'Extract git-tag from repository'
     default: "false"
     required: false
+  dockerfile:
+    description: 'Dockerfile name'
+    default: "Dockerfile"
```

Let's make our new argument optional
```suggestion
    default: "Dockerfile"
    required: false
```